input/LimitStateDesign.tex

```diff
@@ -226,7 +244,7 @@ \section{Stress-Strain Diagrams for Materials}
 
 When the neutral axis falls below the section, the expressions for the
 area (A) of the concrete stress diagram and the location of its centre
-of gravity $(\bar x)$ are given below (\fig 1.5).
+of gravity $(\bar x)$ are given below (\fig \ref{fig:neturalAxis}).
```

Comment:
Shouldn't be "neutralAxis" instead? Also, correct other occurrences of the same.